### DISCUSSION TITLE
Migrate to another compatibility

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#10776] [Request Help]
Can I use Renovate to monitor and switch to new versions with a different "compatibility" tag on the end of the version?

We're currently relying on a version `11.0.10_9-jre-rootless-alpine`, but now that all versions are Alpine, we want to drop the last 7 chars of the tag, and publish simply as `11.0.10_9-jre-rootless`. Can Renovate do that switch for me?

I noticed there is the `allowedVersion` on Package Rules, which name suggests it might help me, but I'm guessing that only works to narrow, not to widen?

I tried this, but it doesn't seem to do what I want:
```js
{
  matchDatasources: ['docker'],
  matchPackageNames: [...],
  matchCurrentVersion: '/.*\\d+-jre-rootless-alpine$/',
  allowedVersions: '/.*\\d+-jre-rootless$/'
},
```

**Comment by rarkins** [comment]
If you temporarily try a different versioning for that package,
Eg loose or maybe maven then it's possible Renovate will sort the new tag as newer and therefore propose it as an *upgrade*
